<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapObjectDialog</class>
 <widget class="QDialog" name="MapObjectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QComboBox" name="ObjectSelectionBox">
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="ObjectGroupBox">
     <property name="title">
      <string>Object</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="ObjectFileLabel">
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="text">
         <string>Image-file path</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="ObjectRoleLabel">
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="text">
         <string>Role</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="TooltipLabel">
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="text">
         <string>Tooltip</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="PositionLabel">
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="text">
         <string>Position</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QSpinBox" name="XSpinBox"/>
      </item>
      <item row="3" column="1" colspan="6">
       <widget class="QLineEdit" name="TooltipLineEdit"/>
      </item>
      <item row="2" column="1" colspan="6">
       <widget class="QComboBox" name="ObjectRoleBox"/>
      </item>
      <item row="0" column="1" colspan="6">
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="10,0">
        <item>
         <widget class="QLabel" name="ObjectFilePathLabel">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="text">
           <string notr="true">[object-image path]</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="BrowseObjectFileButton">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="6" column="3">
       <widget class="QLabel" name="YLabel">
        <property name="text">
         <string>Y:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="4">
       <widget class="QSpinBox" name="YSpinBox"/>
      </item>
      <item row="6" column="5">
       <widget class="QLabel" name="ZLabel">
        <property name="text">
         <string>Z:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="6">
       <widget class="QDoubleSpinBox" name="ZSpinBox"/>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="XLabel">
        <property name="text">
         <string>X:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MapObjectDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MapObjectDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>MapObjectDialog</receiver>
   <slot>buttonClick(QAbstractButton*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>409</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BrowseObjectFileButton</sender>
   <signal>clicked()</signal>
   <receiver>MapObjectDialog</receiver>
   <slot>setObjectFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>481</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ObjectSelectionBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>MapObjectDialog</receiver>
   <slot>changeItem()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setChanges()</slot>
  <slot>buttonClick(QAbstractButton*)</slot>
  <slot>setObjectFile()</slot>
  <slot>changeItem()</slot>
 </slots>
</ui>
